docker/base.dockerfile

```diff
@@ -22,6 +22,7 @@ RUN yum update -y \
   && yum install -y python python-pip python-devel postgresql openssl gcc-c++ \
   && pip install -r requirements.txt \
   && yum autoremove -y gcc-c++ python-pip python-devel \
+  && rm -rf /root/.cache/pip \
```

Comment:
Nit: `/root/.cache` or even `/root/` might be good :)

Reply:
Not sure I'm comfortable removing everything under root. Removing the pip cache here saves about 22MB.